README.md

```diff
@@ -1,10 +1,16 @@
-# angular-testcafe [![Build Status](https://travis-ci.org/politie/angular-testcafe.svg?branch=master)](https://travis-ci.org/politie/angular-testcafe)
+# angular-testcafe [![Tests - E2E](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml/badge.svg)](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml)
+
 A custom Angular builder for [TestCafe](http://devexpress.github.io/testcafe/).
 Serves the Angular application, and then runs the TestCafe tests.
 
+# ⚠️**angular-testcafe supersedes @politie/angular-testcafe-builder**⚠️
+Original [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transfered repository and related code, assets and rights to DevExpress. It means that now it is officially maintained and supported under the name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) by original TestCafe authors - the TestCafe team. 
```

Comment:
```suggestion 
[@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transferred the repository, related code, assets, and rights to DevExpress. The TestCafe team maintains and supports the package under the new name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe). 
```

Reply:
Additionally, the @politie/angular-testcafe module in npmjs.com can be set as deprecated: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions and referred to the new version

Reply:
@timnederhoff, thank you so much for everything that you do for the testing community and TestCafe, in particular 👍 We would really like to notify existing users about the new version via `npm deprecate` if you and your former colleagues don't mind it. Unfortunately, we'll have to bother you again to complete this task.

Reply:
Once you've cut a new release l, I'll mark the existing package as deprecated in npm.

Reply:
Thank you so much, I just published `angular-testcafe@2.0.6`. After some thinking, I decided to continue original versioning to preserve existing tags and releases in the repository.

Reply:
@AndreyBelym @timnederhoff I've deprecated https://www.npmjs.com/package/@politie/angular-testcafe-builder in the NPM registry.